.github/workflows/sonar-scan.yml

```diff
@@ -0,0 +1,97 @@
+name: SonarQube Scan
```

Comment:
Is there a way to centralize this workflow so each project doesn't need to define it? (e.g., pull a list of projects and scan them in a central place).

Reply:
Unfortunately, not at the moment. The main reason, we have different workflows across our projects, different numbers of triggered pipelines, and variously named artifacts produced as a result of tests/linters runs. So as soon as we get to the point of a more unified/formalized project structure, your great improvement idea could take place.